Source/Core/VideoCommon/BPMemory.h

```diff
@@ -512,7 +587,69 @@ struct fmt::formatter<TevStageCombiner::AlphaCombiner>
   template <typename FormatContext>
   auto format(const TevStageCombiner::AlphaCombiner& ac, FormatContext& ctx)
   {
-    return format_to(ctx.out(),
+    auto out = ctx.out();
+    if (ac.bias != TevBias::Compare)
+    {
+      // Generate an equation view, simplifying out addition of zero and multiplication by 1
+      // dest = (d (OP) ((1 - c)*a + c*b) + bias) * scale
+
+      // Note that this is simplified with regards to how lerping works; see PixelShaderGen's
+      // WriteTevRegular for more details.
+
+      const bool has_d = ac.d != TevAlphaArg::Zero;
+      const bool has_a = ac.a != TevAlphaArg::Zero;
+      // If either b or c is zero, b*c is zero
+      const bool has_b = ac.b != TevAlphaArg::Zero && ac.c != TevAlphaArg::Zero;
+      const bool has_bias = ac.bias != TevBias::Zero;  // != Compare is already known
+      const bool has_scale = ac.scale != TevScale::Scale1;
+
+      const char op = (ac.op == TevOp::Sub ? '-' : '+');
+
+      if (ac.dest == TevOutput::Prev)
+        out = format_to(out, "dest = ");
+      else
+        out = format_to(out, "{:n} = ", ac.dest);
+
+      if (has_scale)
+        out = format_to(out, "(");
+      if (has_d)
+        out = format_to(out, "{:n}", ac.d);
```

Comment:
It doesn't need it; the names used by `TevAlphaArg` are `"prev", "c0", "c1", "c2", "tex", "ras", "konst", "ZERO"`.  Of these, only "ZERO" is bad, but any case where zero would show up is instead special-cased to exclude the term.

Reply:
I was thinking it could be worth it to have `prev.a`, `c0.a`, `c1.a`, `c2.a` `ras.a` and `konst.a` for consistency with color combiner.

Reply:
I've manually added `.a` to the format strings instead (and added a comment).  Partially, this avoids redundant data, but also, using `alt_names` with the `lerp` change results in this rather bothersome automatic formatting:

```C++
        if (has_a && has_b)
        {
          out =
              format_to(out, "lerp({}, {}, {})", alt_names[ac.a], alt_names[ac.b], alt_names[ac.c]);
        }
```

whereas this is what is obtained otherwise:

```C++
        if (has_a && has_b)
        {
          out = format_to(out, "lerp({:n}.a, {:n}.a, {:n}.a)", ac.a, ac.b, ac.c);
        }
```